src/Monitor/Autoscale.Autorest/docs/New-AzAutoscaleProfileObject.md

```diff
@@ -1,7 +1,7 @@
 ---
 external help file:
 Module Name: Az.Monitor
-online version: https://learn.microsoft.com/powershell/module/az.Autoscale/new-AzAutoscaleProfileObject
+online version: https://docs.microsoft.com/powershell/module/az.Autoscale/new-AzAutoscaleProfileObject
```

Comment:
Hi @VeryEarly , please pull latest generation and regenerate your code to let  this value 'learn'